study-builder/fdahpStudyDesigner/src/main/webapp/WEB-INF/view/study/questionnaire.jsp

```diff
@@ -181,16 +182,16 @@
           <input type="hidden" id="actionTypeForQuestionPage"
                  name="actionTypeForQuestionPage">
           <div class="gray-xs-f mb-xs">
-            Activity Short Title or Key (1 to 50 characters)
+            Activity short title or key (1 to 50 characters)
             <span
                 class="requiredStar">*
             </span>
             <span
                 class="ml-xs sprites_v3 filled-tooltip" data-toggle="tooltip"
-                title="A human readable step identifier and must be unique across all activities of the study.Note that this field cannot be edited once the study is Launched."></span>
+                title="A human-readable identifier that must be unique across all activities of the study. Allowed characters are lowercase alphabets (a-z), digits (0-9), _ (underscore) and -(minus)."></span>
```

Comment:
alphabets is not the correct term for this

Reply:
Fixed review comment

Reply:
@SR-btc please help with this - we should be using common terms like letters and numbers

Reply:
@jkrogers18 we have pushed the changes just now, updated the message to:
"A human-readable identifier that must be unique across all activities of the study. Allowed characters are lowercase letters (a-z), digits (0-9), _ (underscore) and - (minus)."